### PR TITLE
Added fullfacet call on selected filter

### PR DESCRIPTION
**Pull request by signelindstrom**
When filter is selected, the fullfacet endpoint is called again to only show relevant options for current selected filter

### CODE REVIEW

File: src/components/search/filters/FetchFullfacet.tsx [hunk 1]
```diff
@@ -14,16 +14,27 @@ export interface Data {
...
+    selectedFilters: string[];
 }
 
-const FetchFullfacet: React.FC<FetchDataProps> = ({ children }) => {
+const FetchFullfacet: React.FC<FetchDataProps> = React.memo(({ children, selectedFilters }) => {
     const [data, setData] = useState<Data[]>([]);
+    const [url, setUrl] = useState<string>('http://localhost:3000/api/v3/datasets/fullfacet?facets=["type","creationLocation","ownerGroup","keywords"]&fields={}');
+
+    useEffect(() => {
+        let updatedUrl = 'http://localhost:3000/api/v3/datasets/fullfacet?facets=["type","creationLocation","ownerGroup","keywords"]&fields={}';
```
Comment (Junjiequan) [inline src/components/search/filters/FetchFullfacet.tsx:25]: I see the url is hard coded several places. The `http://localhost:3000/api/v3`  should be in one place and can be configured, as we know for sure we need to change it later.

**Review by Junjiequan — CHANGES_REQUESTED** [review]
Left a small comment. 
Other than that my suggestion is to start with state management as soon as possible. Currently you are managing data fetch in every component. It's fine for now, as only fullquery and fullfacet request is used. But later it will become much harder to manage, especially when you need to make request to more than 2 endpoints or even more within the same service.

**Review by Junjiequan — APPROVED** [review]
Left one comment, otherwise looks fine to me

File: config.tsx [hunk 2]
```diff
@@ -0,0 +1 @@
+export const BASE_URL = 'http://localhost:3000/api/v3/'
```
[the remark this answers is not in the document]
Comment (signelindstrom) [reply config.tsx:1]: Yes, you are right!

**Pull request merged**